### PR TITLE
Final Capstone El Dorado-Frontend UI

### PR DESCRIPTION
**Pull request by michael-duke**
## Final Capstone El Dorado (FRONT-END)

- [Project Requirements](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md)
- [Kanban Board](https://github.com/michael-duke/ElDorado-Frontend/projects/1)

### Live Demo

- :smile: Deployed [El Dorado](https://eldorado-rentals.netlify.app/)
### Back-end repo:
- [ Rails Backend](https://github.com/michael-duke/ElDorado-Backend)
- 🌵  [PR](https://github.com/michael-duke/ElDorado-Backend/pull/29)

### Authors: 

- [Aamir Khan](https://github.com/aamirkhan2478) 🥇 
- [Abel Gebeyehu](https://github.com/AbelG101) 🦅 
- [Michael Mesfin](https://github.com/michael-duke)  🚀 
- [Milen Alebel](https://github.com/milen-ale) 😄 

## ⚠️ Before running App
- Make sure to run `npm i` 
- Then run `npm start`
- Run tests using: `npm test`
- Finally,  visit `http://localhost:5173/`
- Testing Users 
```
- email: 'obiwan@jedi.com', password: 'obiwan',
- email: 'asoka@jedi.com', password: 'obiwan',
```

### CODE REVIEW

File: src/pages/ReservationPage.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Reservation from '../components/Reservation';
+
+function ReservationPage() {
+  return (
+    <Reservation />
+  );
+}
```
Comment (Meltrust) [inline src/pages/ReservationPage.jsx]: - [ ]  Please, kindly refactor this function to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const ReservationPage= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

That way, your app will comply with latest standards 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @michael-duke,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

- Beautiful, gorgeous frontend✔️ 
- Tests are passing ✔️ 
- Nice validations✔️



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._

_I strongly recommend you take them into account as they can improve your final evaluation._





Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: src/pages/ReservationPage.jsx (hunk 1 above)
Comment (michael-duke) [reply src/pages/ReservationPage.jsx]: Awesome 🚀 , we will make these changes and make a second review

**Review by Meltrust — APPROVED** [review]
#### Hi @michael-duke,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- Beautiful, gorgeous frontend✔️ 
- `ReservationPage` functional component has been refactored to ES6✔️ 
- The app works as intended✔️
- Nice validations. ✔️ 
- This is one of the top ten projects I've reviewed here at Microverse, and I've reviewed hundreds. ✔️✔️✔️✔️✔️ 
- Congratulations, you've nailed this project. ✔️ 

Cheers and Happy coding!👏👏👏


------

**Comment by michael-duke** [comment]
We want to thank you for your feedback @Meltrust, your PR messages are constructive and we made our App better
See you in the future 🚀 😄

**Pull request closed without merging**